File: Documentation/API/Utilities/ConvertSecondsToTicks.cs
using System;
using RhythmGameUtilities;

const int seconds = 5;
const int resolution = 192;

var tempoChanges = new Tempo[]
{
    new() { Position = 0, BPM = 88000 }, new() { Position = 3840, BPM = 112000 },
    new() { Position = 9984, BPM = 89600 }, new() { Position = 22272, BPM = 112000 },
    new() { Position = 33792, BPM = 111500 }, new() { Position = 34560, BPM = 112000 },
    new() { Position = 42240, BPM = 111980 }
};

var ticks = Utilities.ConvertSecondsToTicks(seconds, resolution, tempoChanges);

Console.WriteLine(ticks); // 1408

File: Documentation/API/Utilities/ConvertSecondsToTicks.lua
---@type RhythmGameUtilities
local rhythmgameutilities = require("rhythmgameutilities")

local seconds = 5;
local resolution = 192;

local tempo_changes = {
    { position = 0,     bpm = 88000 },
    { position = 3840,  bpm = 112000 },
    { position = 9984,  bpm = 89600 },
    { position = 22272, bpm = 112000 },
    { position = 33792, bpm = 111500 },
    { position = 34560, bpm = 112000 },
    { position = 42240, bpm = 111980 }
}

local ticks =
    rhythmgameutilities.convert_seconds_to_ticks(seconds, resolution, tempo_changes);

print(ticks); --1408
